[chart/solr] Description: Updated Version and Readme

roachmd opened the pull request:
Bumped Version 1.1.1
	Bumped AppVersion 7.7.2
	Updated Readme with examples

 On branch master
 Your branch is ahead of 'origin/master' by 22 commits.
   (use "git push" to publish your local commits)

 Changes to be committed:
	modified:   Chart.yaml
	modified:   README.md
	modified:   values.yaml

<!--
Thank you for contributing to helm/charts. Before you submit this PR we'd like to
make sure you are aware of our technical requirements and best practices:

* https://github.com/helm/charts/blob/master/CONTRIBUTING.md#technical-requirements
* https://github.com/helm/helm/tree/master/docs/chart_best_practices

For a quick overview across what we will look at reviewing your PR, please read
our review guidelines:

* https://github.com/helm/charts/blob/master/REVIEW_GUIDELINES.md

Following our best practices right from the start will accelerate the review process and
help get your PR merged quicker.

When updates to your PR are requested, please add new commits and do not squash the
history. This will make it easier to identify new changes. The PR will be squashed
anyways when it is merged. Thanks.

For fast feedback, please @-mention maintainers that are listed in the Chart.yaml file.

Please make sure you test your changes before you push them. Once pushed, a CircleCI
will run across your changes and do some initial checks and linting. These checks run
very quickly. Please check the results. We would like these checks to pass before we
even continue reviewing your changes.
-->

#### What this PR does / why we need it:

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Special notes for your reviewer:

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/chart]`)

w2g-roachmd committed:
Description
	Bumped Version 1.1.1
	Bumped AppVersion 7.7.2
	Updated Readme with examples

 On branch master
 Your branch is ahead of 'origin/master' by 22 commits.
   (use "git push" to publish your local commits)

 Changes to be committed:
	modified:   Chart.yaml
	modified:   README.md
	modified:   values.yaml

Signed-off-by: Michael D Roach <michael.r@ware2go.co>

maorfr commented on incubator/solr/Chart.yaml:
```diff
@@ -2,8 +2,8 @@
 name: "solr"
-version: "1.1.0"
-appVersion: "7.6.0"
+version: "1.1.1"
```
please bump the minor version

maorfr commented on incubator/solr/values.yaml:
```diff
@@ -22,12 +22,12 @@ terminationGracePeriodSeconds: 180
 # Solr pod liveness
 livenessProbe:
-  initialDelaySeconds: 20
+  initialDelaySeconds: 45
```
why is this required?

w2g-roachmd replied:
Running this on my 16GB mac in minikube the start times were longer.

w2g-roachmd committed:
Description:
	Bumped Minor version instead of Patch

 On branch master
 Your branch is up to date with 'origin/master'.

 Changes to be committed:
	modified:   Chart.yaml

Signed-off-by: Michael D Roach <michael.r@ware2go.co>
